pdpipe/core.py

```diff
@@ -191,6 +193,7 @@ def __init__(self, exraise=True, exmsg=None, desc=None, prec=None,
         self._prec_arg = prec
         self._skip = skip
         self._appmsg = '{}..'.format(desc)
```

Comment:
should the label be used here if defined and no desc rather than DEF_DESCRIPTION?

Reply:
I think the label should be added on the beginning of the application message. Something like 
```
self._appmsg = '{}{}..'.format(f'{label}: ' if label else '', self.,desc)
```

Reply:
```python
...
self._appmsg = f'{label}: {desc}')
```

Reply:
No, stages with no supplied `name` shouldn't have an application message such as `": Dropping mostly NaN columns"` (which is what the your current code does), but rather simply `"Dropping mostly NaN columns".`

Please fix to `self._appmsg = '{}{}..'.format(f'{label}: ' if label else '', self.,desc)`